src/views/Discover.jsx

```diff
@@ -6,15 +6,17 @@ import { FaDog, FaBone } from 'react-icons/fa';
 import axios from 'axios';
 
 import CardStack from '../components/Discover/CardStack';
+import useUserContext from '../hooks/useUserContext';
 import './discover.css';
 
 // eslint-disable-next-line react/function-component-definition
 export default function Discover() {
   const [users, setUsers] = useState([]);
   const [loading, setLoading] = useState(false);
 
+  const userContext = useUserContext();
```

Comment:
if you are wanting to pull a particular state or func from the context file write 
const { /* state or func name*/} = useUserContext();